org.eclipse.scanning.sequencer/src/org/eclipse/scanning/sequencer/RunnableDeviceServiceImpl.java

```diff
@@ -275,17 +275,17 @@ public final IPositioner createPositioner() throws ScanningException {
 		}
 		return device;
 	}
-	
+
 	private <T> IRunnableDevice<T> createDevice(T model) throws ScanningException, InstantiationException, IllegalAccessException, URISyntaxException, UnknownHostException {
-		
+
 		final IRunnableDevice<T> scanner;
-		
+
 		if (modelledDevices.containsKey(model.getClass())) {
 			@SuppressWarnings("unchecked")
 			final Class<IRunnableDevice<T>> clazz = (Class<IRunnableDevice<T>>)modelledDevices.get(model.getClass());
 			if (clazz == null) throw new ScanningException("The model '"+model.getClass()+"' does not have a device registered for it!");
 			scanner = clazz.newInstance();
-			
+
 			// TODO Might have other extension point driven devices
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)